test/e2e/benchmark/throughput.go

```diff
@@ -139,6 +140,17 @@ func TwoNodeBigBlock8MB(logger *log.Logger) error {
 	return runBenchmarkTest(logger, "TwoNodeBigBlock8MB", manifest)
 }
 
+func TwoNodeBigBlock8MBV3(logger *log.Logger) error {
+	manifest := bigBlockManifest
+	manifest.MaxBlockBytes = 8 * testnet.MB
+	// HACKHACK: use a version of celestia-app built from a commit on this PR.
+	// This can be removed after the PR is merged to main.
+	manifest.CelestiaAppVersion = "pr-3882"
+	manifest.TxClientVersion = "pr-3882"
+	manifest.GenesisAppVersion = v3.Version
+	return runBenchmarkTest(logger, "TwoNodeBigBlock8MBV3", manifest)
+}
```

Comment:
@staheri14 is this just due this PR being created before the build tags? we can remove this, correct?

Reply:
Are you referring to the version "pr-3882" used for validator? it points to the celestia-app image built based on the current PR, and the reason we had this test was to ensure we can produce 8MB blocks with the new versioned timeouts, as in e2e benchmark tests, we measure the size of blocks when the test is concluded.